docs/badges/processing.rst

```diff
@@ -0,0 +1,39 @@
+Processing
+==========
+
+**Processing** is a process of:
+
+- Open edX `public signals` subscription
+- badge template requirements analysis
+- badge fulfillments update
+
+.. note::
+    Events async processing happens in a separate specific (out of the standard Django request-response flow) worker process.
+    See EventBus documentation for details.
```

Comment:
Do you think it's worth linking to Event Bus documentation? https://openedx.atlassian.net/wiki/spaces/AC/pages/3127869624/Event+Bus+Project